File: SimpleCalculator/app/src/main/java/com/melihakkose/simplecalculator/MainActivity.java
package com.melihakkose.simplecalculator;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    EditText data1;
    EditText data2;
    TextView resultText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        data1 =findViewById(R.id.dataText1);
        data2 =findViewById(R.id.dataText2);
        resultText=findViewById(R.id.resultText);
    }

    public void sum(View view){
        if(data1.getText().toString().matches("") || data2.getText().toString().matches("")){
            resultText.setText("Enter Number!");
        }
        else{
            int number1=Integer.parseInt(data1.getText().toString());
            int number2=Integer.parseInt(data2.getText().toString());

            int result=number1+number2;
            resultText.setText("Result:"+result);
        }

    }
    public void multiply(View view){
        if(data1.getText().toString().matches("") || data2.getText().toString().matches("")){
            resultText.setText("Enter Number!");
        }
        else{
            int number1=Integer.parseInt(data1.getText().toString());
            int number2=Integer.parseInt(data2.getText().toString());

            int result=number1*number2;
            resultText.setText("Result:"+result);
        }

    }
    public void deduct(View view){

        if(data1.getText().toString().matches("") || data2.getText().toString().matches("")){
            resultText.setText("Enter Number!");
        }
        else{
            int number1=Integer.parseInt(data1.getText().toString());
            int number2=Integer.parseInt(data2.getText().toString());

            int result=number1-number2;
            resultText.setText("Result:"+result);
        }
    }
    public void divide(View view){

        if(data1.getText().toString().matches("") || data2.getText().toString().matches("")){
            resultText.setText("Enter Number!");
        }
        else{
            int number1=Integer.parseInt(data1.getText().toString());
            int number2=Integer.parseInt(data2.getText().toString());

            int result=number1/number2;
            resultText.setText("Result:"+result);
        }
    }


}

File: KotlinCalculator/app/src/main/java/com/melihakkose/kotlincalculator/MainActivity.kt
package com.melihakkose.kotlincalculator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var first: Double?=null
    var second: Double?=null
    var result: Double?=null


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun mySum(view: View){
        first=editTextFirst.text.toString().toDoubleOrNull()
        second=editTextSecond.text.toString().toDoubleOrNull()

        if(first==null || second==null){
            textView.text="Error!"
        }else{
            result=first!! +second!!
            textView.text="Result: $result"
        }



    }
    fun myDiv(view:View){
        first=editTextFirst.text.toString().toDoubleOrNull()
        second=editTextSecond.text.toString().toDoubleOrNull()

        if(first==null || second==null){
            textView.text="Error!"
        }else{
            result=first!! /second!!
            textView.text="Result: $result"
        }
    }

    fun myMultiply(view:View){
        first=editTextFirst.text.toString().toDoubleOrNull()
        second=editTextSecond.text.toString().toDoubleOrNull()

        if(first==null || second==null){
            textView.text="Error!"
        }else{
            result=first!! *second!!
            textView.text="Result: $result"
        }

    }
    fun mySub(view:View){
        first=editTextFirst.text.toString().toDoubleOrNull()
        second=editTextSecond.text.toString().toDoubleOrNull()

        if(first==null || second==null){
            textView.text="Error!"
        }else{
            result=first!! -second!!
            textView.text="Result: $result"
        }

    }
}